back/spec/acceptance/location_spec.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'rspec_api_documentation/dsl'
+
+resource 'Location' do
+  explanation 'User request location data.'
+
+  before do
+    header 'Content-Type', 'application/json'
+  end
+
+  get 'web_api/v1/location/autocomplete' do
+    parameter :input, 'Search input', required: true
+    parameter :language, 'Language', required: false
```

Comment:
Internally, we use languages in this format `fr-FR`, but it seems google wants just `fr` https://developers.google.com/maps/faq#languagesupport

Does it work now with other languages? To test it, you can probably comment out `allow` and specify the language like
```ruby
let(:language) { 'fr-FR' }
```

Reply:
I tested it with Dutch and French and it seems to work as expected with our locale formats 🤷‍♀️